src/templates/prototypes/conservative-current.pug

```diff
@@ -2,17 +2,19 @@ extends ../../layouts/layout.pug
 block content
   -var thingPage
 
-  +section--primary
-    //- Page Title
-    h1
-      span Conservative MPs and Lords
-      +link('#', '- :startdate - :enddate parliament')
-      span.lead A to Z
+  .section--primary#content
```

Comment:
If not a thingPage then content ID needs to go the next section, so that it skips the navigation--letter.